packages/currency/src/erc20/index.ts

```diff
@@ -8,12 +8,19 @@ import { supportedNetworks, supportedNetworksDetails, ERC20SymbolDetails } from
  */
 export function getErc20Currency(
   symbol: string,
-  network = 'mainnet',
+  network?: string,
 ): RequestLogicTypes.ICurrency | undefined {
   // Check if it's on one of the other supported networks
-  if (network in supportedNetworks && supportedNetworks[network].has(symbol)) {
+  if (network && network in supportedNetworks && supportedNetworks[network].has(symbol)) {
     return supportedNetworks[network].get(symbol);
   }
+  for (network of Object.keys(supportedNetworks)) {
```

Comment:
if `network` is specified and not found in the previous condition, it should return